swaggerpy/response.py

```diff
@@ -26,9 +26,13 @@ def handle_response_errors(e):
     :raises HTTPError: :class: `swaggerpy.exception.HTTPError`
     """
     args = list(e.args)
+    kwargs = {}
     if hasattr(e, 'response') and hasattr(e.response, 'text'):
```

Comment:
Is there value in setting `kwargs['response']` even though `e.response.text` doesn't exist?